### PR TITLE
Check if event is cleared and show message

### PR DESCRIPTION
**Pull request by jefflau**

### CODE REVIEW

File: src/schema.gql [hunk 1]
```diff
@@ -8,6 +8,7 @@ type Participant {
 type Party {
   name: String
   participants: Int #Big Number
+  balance: Int #Big Number
```
Comment (hiddentao) [inline src/schema.gql]: Actually balance and deposit come back as hex strings

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr61-app-wearekickback.surge.sh

**Comment by makoto** [comment]
@jefflau are you sure it's done? I looked into https://pr61-app-wearekickback.surge.sh/event/0x7d34f72d8728fe5809548655313abfb30a321cd5 but I still saw the withdraw button.
screen shot 2018-11-16 at 20 16 46

@hiddentao Even though I put `message  should appear for those who haven't withdrawn yet`, let's just show to everybody as we don't really need to hide from everybody else and makes our debugging a lot easier, too.

**Comment by jefflau** [comment]
Try a new a event. Past events might not work

**Comment by hiddentao** [comment]
@jefflau We need it to work for past events too. It should work for them.

**Comment by jefflau** [comment]
I've switched to a front-end web3 call for the contract balance, which seems to work. I think it was the server not updating the value of `balance` in the party. I'd vote for getting what we can in the front-end as we have to wait 10 minutes for the back-end. If we decide to just use the backend, we can revert this. The only commit should work if so.

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr61-app-wearekickback.surge.sh

**Comment by jefflau** [comment]
@hiddentao @makoto

File: src/schema.gql (hunk 1 above)
Comment (hiddentao) [reply src/schema.gql]: will fix when we shift all into the `@wearekickback/shared` pkg

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr61-app-wearekickback.surge.sh

**Comment by noblocknoparty-devs** [comment]
PR deployed at: https://pr61-app-wearekickback.surge.sh

**Pull request merged**